Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -315,7 +315,10 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
-	AND(32, R(RSCRATCH), Imm32(~31));
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bWii)
+		AND(32, R(RSCRATCH), Imm32(~31));
+	else
+		AND(32, R(RSCRATCH), Imm32(Memory::RAM_MASK & ~31));
```

Comment:
Anything you do here is likely to get rewritten by dynamic-bat.  Given that, masking before the PXOR isn't too terrible, I guess.